INSTALL.md

```diff
@@ -1,11 +1,28 @@
-# Installing
+# Installation
 
-- [Installing from source](#installing-from-source)
-- [Installing from a package manager](#installing-from-a-package-manager)
+- [Install from a package manager](#install-from-a-package-manager)
+- [Install from source](#install-from-source)
 
-## Installing from source
+## Install from a package manager
 
-The easiest way to install is via the existing generated Scheme code.
+You can install Idris 2 with any one of the following package managers.
+
+Install with [Homebrew](https://brew.sh/) using
+```sh
+brew install idris2
+```
+Install with [Nix](https://nixos.org/features.html) using
+```sh
+nix-env -i idris2
+```
+Install with [Nix Flakes](https://nixos.wiki/wiki/Flakes) using
+```sh
+nix profile install github:idris-lang/Idris2
+```
+
```

Comment:
I think that once you are making installation instructions more accessible, then maybe, installation through [`pack`](https://github.com/stefan-hoeck/idris2-pack/blob/main/INSTALL.md) should be mentioned here also. This is for now a very good alternative for installation, especially if one wants to *just use* the latest Idris, especially with lots of ready libraries.

Reply:
done

Reply:
> done

This is great, thank you! But I'm afraid adding _"Make sure that $HOME/.pack/bin is on your PATH and takes precedence over the bin folder(s) (if any) where existing versions of Idris2 are already installed"_ from this `INSTALL.md` is also important, or else this text could be understood as "run `bash -c ...` and `idris2` command becomes usable", where setting `PATH` is important too.

Reply:
The pack instructions have become more detailed, so I've just decided to link to them instead